dapp/src/components/layout/Navbar/Navbar.tsx

```diff
@@ -54,11 +54,18 @@ export function Navbar() {
                             ))}
                         </div>
                     </div>
+                    <div className="md:hidden">
+                        <NavbarMobile routes={ROUTES}>
+                            <ConnectButton key="connect" />
```

Comment:
I dont see how this key is needed, rest lgtm!
```suggestion
                            <ConnectButton />
```